### PR TITLE
Commenting Fill<T>(T[], T, Int32, Int32) overload

### PR DESCRIPTION
**Pull request by FarhadShariatzadeh**
Added description to parameters and summary.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by FarhadShariatzadeh** [comment]
> Thank you very much for filing this PR. I hate to ask more of you, but could you adjust the wording of the descriptions to match similar wording found in other methods? For example, this method is almost the same (regarding parameter naming)
> 
> https://github.com/dotnet/dotnet-api-docs/blob/fc75f1dabcd41a5efb863f30217ff59c0dce5e11/xml/System/Array.xml#L3230-L3234

Not a problem @Thraka ! Sure, I adjusted the wording to match similar wording in other methods. Please let me know if I need to make any more changes. Thank you.

File: xml/System/Array.xml [hunk 1]
```diff
@@ -2806,12 +2806,13 @@
...
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="array">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="startIndex">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
+        <typeparam name="T">The type of the elements of the array.</typeparam>
+        <param name="array">The <see cref="T:System.Array" /> to be filled.</param>
+        <param name="value">The new <paramref name="value"/> for the elements in the specified range.</param>
```
Comment (Thraka) [inline xml/System/Array.xml]: One last minor fix. The value param doesn't need to reference itself 😄 
```suggestion
        <param name="value">The new value for the elements in the specified range.</param>
```

File: xml/System/Array.xml [hunk 2]
```diff
@@ -2766,10 +2766,10 @@
         <Parameter Name="value" Type="T" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="array">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <typeparam name="T">The type of the elements of the array.</typeparam>
+        <param name="array">The <see cref="T:System.Array" /> to be filled.</param>
+        <param name="value">The new <paramref name="value"/> for the elements in the specified range.</param>
```
Comment (Thraka) [inline xml/System/Array.xml]: One last minor fix. The value param doesn't need to reference itself 😄 
```suggestion
        <param name="value">The new value for the elements in the specified range.</param>
```

**Comment by FarhadShariatzadeh** [comment]
> One last minor fix to apply in both instances.

Of course, thank you for your comments and reviewing my pull request.

**Comment by FarhadShariatzadeh** [comment]
Thank you @rpetrusha  and @Thraka  again for reviewing my changes. I believe I have responded to the changes you mentioned. Would you please let me know if there is anything I need to do at this point?

**Pull request merged**